doc/nrf/app_dev/board_names.rst

```diff
@@ -100,6 +100,10 @@ Also see the :ref:`zephyr:boards` section in the Zephyr documentation.
 |                   |            |                                                     |                                                                       |
 |                   |            |                                                     | ``nrf54h20dk/nrf54h20/cpuppr``                                        |
 +-------------------+------------+-----------------------------------------------------+-----------------------------------------------------------------------+
+| nRF54LM20 DK      | PCA10184   | nrf54lm20dk (TBA)                                   | ``nrf54lm20dk/nrf54lm20/cpuapp``                                      |
```

Comment:
```suggestion
| nRF54LM20 DK      | PCA10184   | zephyr:board:: nrf54lm20dk                                  | ``nrf54lm20dk/nrf54lm20/cpuapp``                                      |
```
Please fix the table if it is distorted with my comment. We have zephyr page here - https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/boards/nordic/nrf54lm20dk/doc/index.html

Reply:
We could as well update the "sample_board_rows.txt" file with the required links and PCA number.